pyproject.toml

```diff
@@ -13,6 +13,8 @@ pytest = "^7.1.2"
 PyYAML = "^6.0"
 black = "^22.3.0"
 flake8 = "^4.0.1"
+isort = "^5.10.1"
+flake8-isort = "^4.1.1"
```

Comment:
Wouldn't these better fit inside `dev-dependencies` below? Which also applies to all other dependencies here besides `python` and `pyyaml`, I think